docs/user/supported_devices.rst

```diff
@@ -21,6 +21,7 @@ ar71xx-generic
 * AVM
 
   - Fritz!Box 4020 [#avmflash]_
+  - Fritz!Box 7312 [#avmflash]_
```

Comment:
This is not the correct place, as the 7312 is not part of the ar71xx-generic target. Add a new section for the lantiq-xway target and add the FRITZ!Box 7312 there.